tests/devices/test_default_qubit.py

```diff
@@ -2406,6 +2406,25 @@ def circuit():
         # evaluated one expval altogether
         assert spy.call_count == 1
 
+    def test_do_not_split_analytic_torch(self, mocker):
```

Comment:
Perhaps out of scope and an overall organizational question: wouldn't these tests belong to the dedicated files for each backprop device? So e.g., in `test_default_qubit_torch.py`?

Reply:
Yes, good point! The other tests are already in this file, however, so it would likely be out of scope to move all tests here.